packages/contracts/contracts/optimistic-ethereum/OVM/bridge/tokens/OVM_L2StandardBridge.sol

```diff
@@ -0,0 +1,220 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >0.5.0 <0.8.0;
+pragma experimental ABIEncoderV2;
+
+/* Interface Imports */
+import { iOVM_L1StandardBridge } from "../../../iOVM/bridge/tokens/iOVM_L1StandardBridge.sol";
+import { iOVM_L1ERC20Bridge } from "../../../iOVM/bridge/tokens/iOVM_L1ERC20Bridge.sol";
+import { iOVM_L2ERC20Bridge } from "../../../iOVM/bridge/tokens/iOVM_L2ERC20Bridge.sol";
+
+/* Library Imports */
+import { ERC165Checker } from "@openzeppelin/contracts/introspection/ERC165Checker.sol";
+import { OVM_CrossDomainEnabled } from "../../../libraries/bridge/OVM_CrossDomainEnabled.sol";
+import { Lib_PredeployAddresses } from "../../../libraries/constants/Lib_PredeployAddresses.sol";
+
+/* Contract Imports */
+import { IL2StandardERC20 } from "../../../libraries/standards/IL2StandardERC20.sol";
+
+/**
+ * @title OVM_L2StandardBridge
+ * @dev The L2 Standard bridge is a contract which works together with the L1 Standard bridge to enable
+ * ETH and ERC20 transitions between L1 and L2.
+ * This contract acts as a minter for new tokens when it hears about deposits into the L1 Standard bridge.
+ * This contract also acts as a burner of the tokens intended for withdrawal, informing the L1 bridge to release L1 funds.
+ *
+ * Compiler used: optimistic-solc
+ * Runtime target: OVM
+ */
+contract OVM_L2StandardBridge is iOVM_L2ERC20Bridge, OVM_CrossDomainEnabled {
+
+    /********************************
+     * External Contract References *
+     ********************************/
+
+    address public l1TokenBridge;
+
+    /***************
+     * Constructor *
+     ***************/
+
+    /**
+     * @param _l2CrossDomainMessenger Cross-domain messenger used by this contract.
+     * @param _l1TokenBridge Address of the L1 bridge deployed to the main chain.
+     */
+    constructor(
+        address _l2CrossDomainMessenger,
+        address _l1TokenBridge
+    )
+        OVM_CrossDomainEnabled(_l2CrossDomainMessenger)
+    {
+        l1TokenBridge = _l1TokenBridge;
+    }
+
+    /***************
+     * Withdrawing *
+     ***************/
+
+    /**
+     * @inheritdoc iOVM_L2ERC20Bridge
+     */
+    function withdraw(
+        address _l2Token,
+        uint256 _amount,
+        uint32, // _l1Gas,
+        bytes calldata _data
+    )
+        external
+        override
+        virtual
+    {
+        _initiateWithdrawal(
+            _l2Token,
+            msg.sender,
+            msg.sender,
+            _amount,
+            0,
+            _data
+        );
+    }
+
+    /**
+     * @inheritdoc iOVM_L2ERC20Bridge
+     */
+    function withdrawTo(
+        address _l2Token,
+        address _to,
+        uint256 _amount,
+        uint32, // _l1Gas,
```

Comment:
We may want to add this variable back into active use in the case that we want to start considering the provided gas parameter when sending messages from L2 => L1. Alternatively, we can just create a ticket for this and do it post-uniswap.

Reply:
@smartcontracts do you plan on making a ticket for this behavior?  I am passing it through now